Update several Persian channels

alr86 opened the pull request:
Update name and logo of several channels, add "TW Sports", remove "aio sports" and...

Alstruit reviewed (changes requested):
Explain the removal of several channels.
The only accepted reasons are:
1. The channel never existed.
2. The channel is a duplicate.

If a channel no longer exist due to a closure, mark the date of closure followed by the replaced id section.

alr86 commented:
I've added three new Telewebion Sport channels with the IDs `TWSport{1-2-3}.ir`

I mistakenly deleted the aiosport channels.  I've realized they were only deleted from the website and are still available in their app (even if they weren't, I should have marked them as closed but anyway 🙄).

Some channels were renamed according to this Wikipedia page:

 - `KermanshahTV.ir` -> `ZagrosTV.ir`
 - `GolestanTV.ir` -> `SabzTV.ir`
 - `IRIB5.ir` -> `TehranTV.ir`
 - `KhorasanShomaliTV.ir` -> `AtrakTV.ir`
 - `KhorasanJonoobiTV.ir` -> `KhavaranTV.ir`

`AlalamNewsChannel.ir` was the same as `AlAlam.ir`

Some IRIB channels were inconsistently saved as XTV.ir and XNetwork.ir over time. For consistency, I've renamed them all to NameTV.ir:

 - `KishNetwork.ir` -> `KishTV.ir`
 - `WestAzerbaijanNetwork.ir` -> `WestAzerbaijanTV.ir`
 - `YazdNetwork.ir` -> `YazdTV.ir`
 - `KhavaranNetwork.ir` -> `KhavaranTV.ir`
 - `JameJamTVNetwork{1-2-3}.ir` -> `JameJamTV{1-2-3}.ir`
 - `BaranNetwork.ir` -> `BaranTV.ir`
 - `AtrakNetwork.ir` -> `AtrakTV.ir`

Alstruit commented on data/channels.csv:
```diff
@@ -38686,7 +38686,6 @@ YateemTV.th,Yateem TV,,,,TH,,,c/TH,tha,,FALSE,,,,https://yateem.tv/,https://i.im
 YATV.et,YA TV,,,,ET,,Addis Ababa,c/ET,amh,entertainment,FALSE,,,,https://www.yaet.tv/,https://i.imgur.com/EJ3vpbk.jpeg
 YaTV.ru,YaTV,ЯТВ,,,RU,,Yar-Sale,c/RU,rus,,FALSE,1992-12-31,,,https://yamaltime.ru/,https://i.imgur.com/wKWhThg.png
 YATVO.ve,YATVO,,,Yara Televisión Online,VE,VE-U,San Felipe Yaracuy,c/VE,spa,entertainment,FALSE,,,,https://www.yatvo.com,https://i.imgur.com/KuDT78V.png
-YazdNetwork.ir,Yazd Network,صدا و سیمای مرکز یزد,,,IR,,Yazd,c/IR,pes,general,FALSE,,,,https://yazd.iribnews.ir/,https://namnamak.com/media/k2/items/cache/9179a7c9b1ded255b3db7ebe2c5d04a6_XL.jpg
```
Where is YazdTV.ir?

alr86 replied:
> Where is YazdTV.ir?

I mistakenly used the old name of the channel, TabanTV, instead of YazdTV.

BellezaEmporium commented:
> I've added three new Telewebion Sport channels with the IDs `TWSport{1-2-3}.ir`
> 

Great addition.


> Some channels were renamed according to [THIS(Persian)](https://fa.wikipedia.org/wiki/%D9%81%D9%87%D8%B1%D8%B3%D8%AA_%D8%B4%D8%A8%DA%A9%D9%87%E2%80%8C%D9%87%D8%A7%DB%8C_%D8%AA%D9%84%D9%88%DB%8C%D8%B2%DB%8C%D9%88%D9%86%DB%8C_%D8%B5%D8%AF%D8%A7_%D9%88_%D8%B3%DB%8C%D9%85%D8%A7%DB%8C_%D8%AC%D9%85%D9%87%D9%88%D8%B1%DB%8C_%D8%A7%D8%B3%D9%84%D8%A7%D9%85%DB%8C_%D8%A7%DB%8C%D8%B1%D8%A7%D9%86) Wikipedia page:
> 
>     * `KermanshahTV.ir` -> `ZagrosTV.ir`
>     * `GolestanTV.ir` -> `SabzTV.ir`
>     * `IRIB5.ir` -> `TehranTV.ir`
>     * `KhorasanShomaliTV.ir` -> `AtrakTV.ir`
>     * `KhorasanJonoobiTV.ir` -> `KhavaranTV.ir`
> 

If they used to be called like that before, please keep the old name as an alt. name, to prevent confusion.
> 
> `AlalamNewsChannel.ir` was the same as `AlAlam.ir`
> 

Noted.

> Some IRIB channels were inconsistently saved as NameTV.ir and NameNetwork.ir over time. For consistency, I've renamed them all to NameTV.ir:
> 
>     * `KishNetwork.ir` -> `KishTV.ir`
>     * `WestAzerbaijanNetwork.ir` -> `WestAzerbaijanTV.ir`
>     * `YazdNetwork.ir` -> `YazdTV.ir`
>     * `KhavaranNetwork.ir` -> `KhavaranTV.ir`
>     * `JameJamTVNetwork{1-2-3}.ir` -> `JameJamTV{1-2-3}.ir`
>     * `BaranNetwork.ir` -> `BaranTV.ir`
>     * `AtrakNetwork.ir` -> `AtrakTV.ir`
>

Agreed.

BellezaEmporium commented:
> I've added three new Telewebion Sport channels with the IDs `TWSport{1-2-3}.ir`
> 

Great addition.


> Some channels were renamed according to [THIS(Persian)](https://fa.wikipedia.org/wiki/%D9%81%D9%87%D8%B1%D8%B3%D8%AA_%D8%B4%D8%A8%DA%A9%D9%87%E2%80%8C%D9%87%D8%A7%DB%8C_%D8%AA%D9%84%D9%88%DB%8C%D8%B2%DB%8C%D9%88%D9%86%DB%8C_%D8%B5%D8%AF%D8%A7_%D9%88_%D8%B3%DB%8C%D9%85%D8%A7%DB%8C_%D8%AC%D9%85%D9%87%D9%88%D8%B1%DB%8C_%D8%A7%D8%B3%D9%84%D8%A7%D9%85%DB%8C_%D8%A7%DB%8C%D8%B1%D8%A7%D9%86) Wikipedia page:
> 
>     * `KermanshahTV.ir` -> `ZagrosTV.ir`
>     * `GolestanTV.ir` -> `SabzTV.ir`
>     * `IRIB5.ir` -> `TehranTV.ir`
>     * `KhorasanShomaliTV.ir` -> `AtrakTV.ir`
>     * `KhorasanJonoobiTV.ir` -> `KhavaranTV.ir`
> 

If they used to be called like that before, please keep the old name as an alt. name, to prevent confusion.
> 
> `AlalamNewsChannel.ir` was the same as `AlAlam.ir`
> 

Noted.

> Some IRIB channels were inconsistently saved as NameTV.ir and NameNetwork.ir over time. For consistency, I've renamed them all to NameTV.ir:
> 
>     * `KishNetwork.ir` -> `KishTV.ir`
>     * `WestAzerbaijanNetwork.ir` -> `WestAzerbaijanTV.ir`
>     * `YazdNetwork.ir` -> `YazdTV.ir`
>     * `KhavaranNetwork.ir` -> `KhavaranTV.ir`
>     * `JameJamTVNetwork{1-2-3}.ir` -> `JameJamTV{1-2-3}.ir`
>     * `BaranNetwork.ir` -> `BaranTV.ir`
>     * `AtrakNetwork.ir` -> `AtrakTV.ir`
>

Agreed.

alr86 commented:
> If they used to be called like that before, please keep the old name as an alt. name, to prevent confusion.

I already did that.

Alstruit reviewed (dismissed):
If there are no further objections... (by me of course).
Edit: Oops said it too soon.
